docs/chapters/04-1.md

```diff
@@ -0,0 +1,218 @@
+---
+title: Linear Algebra and Convolutions
+authors: Yuchi Ge, Anshan He, Shuting Gu, and Weiyang Wen
+---
+
+## [Linear Algebra Review](0:00:00-0:21:26)
+
+This part is a recap of basic linear algebra in the context of neural networks. We start with a simple hidden layer $\vec{h}$:
```

Comment:
Please, use `\vect{}` and add the macro to `https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/_layouts/withmacros.html` such that it renders it as `\boldsymbol{}`.